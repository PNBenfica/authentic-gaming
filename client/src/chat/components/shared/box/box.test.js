import renderer from 'react-test-renderer';
import Box from './box';

it('renders correctly', () => {
  const tree = renderer
    .create(<Box className='my-class'>content</Box>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});