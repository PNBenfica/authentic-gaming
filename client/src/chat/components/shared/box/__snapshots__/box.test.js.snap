// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="box my-class"
>
  content
</div>
`;
